src/sonar_widget/SonarPlot.cc

```diff
@@ -7,9 +7,13 @@ using namespace frame_helper;
 SonarPlot::SonarPlot(QWidget *parent)
     : QFrame(parent), changedSize(true),scaleX(1),scaleY(1),range(5)
 {
+
     // apply default colormap
     applyColormap(COLORGRADIENT_JET);
 
+    numDiv = 640;
```

Comment:
By SonarBeam or Sonar datatype, it is not possible to get/calculate the motor step size angle of scanning sonars. I looked the Micron driver and this value is fixed (https://github.com/rock-drivers/drivers-sonar_tritech/blob/master/src/SeaNetMicron.cpp), so I just replied to SonarWidget as well.